.github/workflows/self-hosted.yml

```diff
@@ -117,101 +98,11 @@ jobs:
-
-          docker manifest create \
-            ghcr.io/getsentry/sentry:latest \
-            --amend ghcr.io/getsentry/sentry:${{ github.sha }}-amd64 \
-            --amend ghcr.io/getsentry/sentry:${{ github.sha }}-arm64
-
-          docker manifest push ghcr.io/getsentry/sentry:latest
-
-          docker manifest create \
-            ghcr.io/getsentry/sentry:nightly \
-            --amend ghcr.io/getsentry/sentry:${{ github.sha }}-amd64 \
-            --amend ghcr.io/getsentry/sentry:${{ github.sha }}-arm64
-
-          docker manifest push ghcr.io/getsentry/sentry:nightly
-
-          # These manifests below are being used for development purposes.
-          # We shall keep it as is.
-          docker manifest create \
-            ghcr.io/getsentry/sentry-self-hosted:${{ github.sha }} \
-            --amend ghcr.io/getsentry/sentry:${{ github.sha }}-amd64 \
-            --amend ghcr.io/getsentry/sentry:${{ github.sha }}-arm64
-
-          docker manifest push ghcr.io/getsentry/sentry-self-hosted:${{ github.sha }}
-
-          docker manifest create \
-            ghcr.io/getsentry/sentry-self-hosted:latest \
-            --amend ghcr.io/getsentry/sentry:${{ github.sha }}-amd64 \
-            --amend ghcr.io/getsentry/sentry:${{ github.sha }}-arm64
-
-          docker manifest push ghcr.io/getsentry/sentry-self-hosted:latest
-
-  publish-dockerhub:
-    needs: [assemble]
-    runs-on: ubuntu-latest
-    if: ${{ (github.ref_name == 'master') }}
-    steps:
-      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
-      - name: Get short SHA for docker tag
-        id: short_sha
-        shell: bash
-        run: |
-          SHORT_SHA=$(git rev-parse --short "$GITHUB_SHA")
-          if [[ -z "$SHORT_SHA" ]]; then
-            echo "Short SHA empty? Re-running rev-parse."
-            git rev-parse --short "$GITHUB_SHA"
-          else
-            echo "sha=$SHORT_SHA" >> $GITHUB_OUTPUT
-          fi
-      - name: Push built docker image
-        shell: bash
-        env:
-          SHORT_SHA: ${{ steps.short_sha.outputs.sha }}
-          IMAGE_URL: ghcr.io/getsentry/sentry-self-hosted:${{ github.sha }}
-        run: |
-          # only login if the password is set
-          if [[ "${{ secrets.DOCKER_HUB_RW_TOKEN }}" ]]; then echo "${{ secrets.DOCKER_HUB_RW_TOKEN }}" | docker login --username=sentrybuilder --password-stdin; fi
-
-          # We push 3 tags to Dockerhub:
-          # first, the full sha of the commit
-          docker buildx imagetools create --tag getsentry/sentry:${GITHUB_SHA} ghcr.io/getsentry/sentry:${{ github.sha }}
-
-          # second, the short sha of the commit
-          docker buildx imagetools create --tag getsentry/sentry:${SHORT_SHA} ghcr.io/getsentry/sentry:${{ github.sha }}
-
-          # finally, nightly
-          docker buildx imagetools create --tag getsentry/sentry:nightly ghcr.io/getsentry/sentry:${{ github.sha }}
+          docker buildx imagetools create \
```

Comment:
This is much simpler than what we had before, tested a dry run here: https://github.com/getsentry/sentry/actions/runs/16687698374/job/47240299657